### PR TITLE
Sorting order by name

### PR DESCRIPTION
**Pull request by BartoszSiejka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #2313
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by kayue** [comment]
5 commits for changing few lines? 
Maybe squash commits into one?

File: src/Sylius/Bundle/WebBundle/Behat/WebContext.php [hunk 1]
```diff
@@ -888,4 +888,23 @@ public function iViewDeletedElements()
...
         $this->clickLink('Show deleted');
     }
-}
+    
+    /**
+     * @Then I should see table sorted by lastName column
+     */
+    public function iShouldSeeTableSortedByLastName()
+    {
+        $allNames = $this->getSession()->getPage()->findAll('css', '#orders > tbody > tr > td > p');
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Behat/WebContext.php]: I think you can make this method more generic by passing a argument to this method which the id of the table.

**Comment by aRn0D** [comment]
@kayue has right, you should squash your commit !

**Comment by BartoszSiejka** [comment]
| Q             | A
| ------------- | ---
| Bug fix?      | [yes|no]
| New feature?  | [yes|no]
| BC breaks?    | [yes|no]
| Deprecations? | [yes|no]
| Fixed tickets | [comma separated list of tickets fixed by the PR]
| License       | MIT
| Doc PR        | [The reference to the documentation PR if any]

File: features/backend/orders.feature [hunk 2]
```diff
@@ -118,3 +118,8 @@ Feature: Orders management
     Scenario: Displaying correct tax total on order page
         Given I am viewing order with number "000000002"
          Then I should see "Tax total: €10.58"
+
+    Scenario: Sorting order tabel by appropriate column
```
Comment (amenophis) [inline features/backend/orders.feature]: tabel => table

**Comment by pjedrzejewski** [comment]
Thanks Bartek! We should make this more generic later!

**Pull request merged**